crwn-clothing/src/routes/categories-preview/categories-preview.component.jsx

```
@@ -1,19 +1,23 @@
     <Fragment>
-      {Object.keys(categoriesMap).map(title => 
-        <Fragment>
-            {Object.keys(categoriesMap).map((title) => {
-              const products = categoriesMap[title];
-              return <CategoryPreview key={title} title={title} products={products} />;
-            })}
-        </Fragment>
-      )}
+      {
+        isLoading?<Spinner/>:Object.keys(categoriesMap).map(title => 
+          <Fragment>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)